test/e2e/service_test.go

```diff
@@ -106,34 +106,47 @@ func (data *TestData) createClusterIPServiceAndBackendPods(t *testing.T, name st
 	return svc, cleanup
 }
 
-// TestNodePortWindows tests NodePort Service on Windows Node. It is a temporary test to replace upstream Kubernetes one:
-// https://github.com/kubernetes/kubernetes/blob/ea0764452222146c47ec826977f49d7001b0ea8c/test/e2e/windows/service.go#L42
-// Issue: https://github.com/antrea-io/antrea/issues/2289
-func TestNodePortWindows(t *testing.T) {
-	skipIfNoWindowsNodes(t)
-
+// TestNodePort tests traffic from Pods to NodePort Service.
+func TestNodePort(t *testing.T) {
```

Comment:
should this change belong in a separate PR?

Reply:
I updated the commit message to focusing on NodePort Service test. Since the change is small, I prefer to put them together.

Reply:
sounds good